CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Governance proposals can be submitted successfully against secondaries (#2247)
 - `set_ca_cert`/`remove_ca_cert` proposals have been renamed `set_ca_cert_bundle`/`remove_ca_cert_bundle` and now also accept a bundle of certificates encoded as concatenated PEM string (#2221).
 
+### Added
+
+- Support for multiple key wrapping algorithms for C++ and JavaScript applications (#2246)
```

Comment:
Sphinx has a js domain (https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-javascript-domain) as we expand our JS API it would be good to make use of it to document what we provide.